index.js

```diff
@@ -1,11 +1,26 @@
-const express = require("express");
+const express = require('express');
 const app = express();
-require("dotenv").config();
+const bodyParser = require('body-parser');
+const db = require('./controllers');
+const port = 8000;
+const cors = require('cors');
```

Comment:
Pas la peine d'activer CORS côté serveur. Côté client, il faudra régler un proxy dans `package.json` : https://create-react-app.dev/docs/proxying-api-requests-in-development.

Reply:
merci pour le lien!